sql-join/big-spenders.sql

```diff
@@ -0,0 +1,5 @@
+select "firstName",
+      "lastName"
+  from "customers"
+  join "payments" using ("customerId")
+  limit 10;
```

Comment:
This should be the 10 _largest_ payments, not just any 10:

---

![image](https://user-images.githubusercontent.com/16143771/144539060-87b814fe-84a7-4384-ac46-8a4d317d4de5.png)